src/DotNetLightning.Core/Channel/Commitments.fs

```diff
@@ -187,6 +194,44 @@ type Commitments = {
             | Some _, Some htlcIn -> htlcIn.Add |> Some
             | _ -> None
 
+        member this.RemoteCommitAmount(): Amounts= 
+            let commitFee = Transactions.commitTxFee 
+                                this.RemoteParams.DustLimitSatoshis 
+                                this.RemoteCommit.Spec
+            
+            let (toLocalAmount, toRemoteAmount) =
+                if (this.LocalParams.IsFunder) then
+                    (this.RemoteCommit.Spec.ToLocal.Satoshi
+                     |> Money.Satoshis),
+                    (this.RemoteCommit.Spec.ToRemote.Satoshi
+                     |> Money.Satoshis) - commitFee 
+                else
+                    (this.RemoteCommit.Spec.ToLocal.Satoshi
+                     |> Money.Satoshis) - commitFee,
+                    (this.RemoteCommit.Spec.ToRemote.Satoshi
+                     |> Money.Satoshis)
+
+            {Amounts.ToLocal = toLocalAmount; ToRemote = toRemoteAmount}
+
+        member this.LocalCommitAmount(): Amounts= 
+            let commitFee = Transactions.commitTxFee 
+                                this.LocalParams.DustLimitSatoshis 
+                                this.LocalCommit.Spec
+            
+            let (toLocalAmount, toRemoteAmount) =
+                if (this.LocalParams.IsFunder) then
+                    (this.LocalCommit.Spec.ToLocal.Satoshi
+                     |> Money.Satoshis),
+                    (this.LocalCommit.Spec.ToRemote.Satoshi
+                     |> Money.Satoshis) - commitFee 
```

Comment:
it looks like the fee is being subtracted from the wrong amount here. ie. it should be taken from the `ToLocal` output instead. Similarly for the `else` clause below.